packages/happy-dom/src/nodes/parent-node/ParentNodeUtility.ts

```diff
@@ -190,4 +190,29 @@ export default class ParentNodeUtility {
 
 		return null;
 	}
+
+	/**
+	 * Returns an element by Name.
+	 *
+	 * @param parentNode
+	 * @param name Name.
+	 * @returns Matching element.
+	 */
+	public static getElementsByName(
```

Comment:
This function can be moved to Document.getElementsByName().

Reply:
recursion function, i will try to move it in my power.